graphql/schema/mutation.graphql

```diff
@@ -1,33 +1,33 @@
 # This file lists all of the mutations. The mutation definitions can be found in the corresponding files in the resolvers folder.
 type Mutation {
   # annotations
-  bbCreateTicket(taskId: String!, execution: Int): Boolean!
+  bbCreateTicket(taskId: String! @requireProjectAccess(permission: ANNOTATIONS, access: EDIT), execution: Int): Boolean!
   addAnnotationIssue(
-    taskId: String!
+    taskId: String! @requireProjectAccess(permission: ANNOTATIONS, access: EDIT)
     execution: Int!
     apiIssue: IssueLinkInput!
     isIssue: Boolean!
   ): Boolean!
   editAnnotationNote(
-    taskId: String!
+    taskId: String! @requireProjectAccess(permission: ANNOTATIONS, access: EDIT)
     execution: Int!
     originalMessage: String!
     newMessage: String!
   ): Boolean!
   moveAnnotationIssue(
-    taskId: String!
+    taskId: String! @requireProjectAccess(permission: ANNOTATIONS, access: EDIT)
     execution: Int!
     apiIssue: IssueLinkInput!
     isIssue: Boolean!
   ): Boolean!
   removeAnnotationIssue(
-    taskId: String!
+    taskId: String! @requireProjectAccess(permission: ANNOTATIONS, access: EDIT)
     execution: Int!
     apiIssue: IssueLinkInput!
     isIssue: Boolean!
   ): Boolean!
   setAnnotationMetadataLinks(
-    taskId: String!
+    taskId: String! @requireProjectAccess(permission: ANNOTATIONS, access: EDIT)
     execution: Int!
     metadataLinks: [MetadataLinkInput!]!
   ): Boolean!
```

Comment:
Can't comment on these since they were added earlier but should the delete / save distro mutations also requireDistroAccess?

Reply:
(Same comment as below; they are restricted, but you have to go down into other files to see the directive)